### PR TITLE
Update advanced compass information

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-compass-setup-advanced.rst [hunk 1]
```diff
@@ -33,17 +33,28 @@ Compass Settings
...
+   Mission Planner: Advanced Compass Setup and Calibration
 
-Compass Priority
-----------------
 
-All the attached compasses that the autopilot discovered during initialization are shown in the upper area of the page. You can determine the ordering, from primary on down, by using the arrow keys next to the compass. The first in the list is will be the one used, unless there is a problem. In which case, either the second or third listed, if enabled, will be used, automatically determined by the autopilot as to which is giving the best readings.
+Compass Ordering and Priority
+-----------------------------
+
+During boot, ArduPilot automatically detects the compasses present in the system, adds them to a list, and assigns the first three a priority (1-3) linked to their DEV ID (``COMPASS_PRIORx_ID``), according to the order in which they are discovered. This priority determines which compass is used by the EKF lanes. The primary compass (highest priority, 1) will be used by all lanes, and fallback to the next viable compass in the first three priorities, if the primary becomes unhealthy.
```
Comment (bugobliterator) [inline common/source/docs/common-compass-setup-advanced.rst:47]: COMPASS_PRIOx_ID not COMPASS_PRIORx_ID

File: common/source/docs/common-compass-setup-advanced.rst [hunk 2]
```diff
@@ -33,17 +33,28 @@ Compass Settings
...
+
+The list of discovered compasses and their priority is maintained across boots.
+
+If a user wishes to change to a different compass for one of the three priorities, he can change the ``COMPASS_PRIORx_ID`` to that compass's ``COMPASS_DEV_IDx`` ID value. If a ``COMPASS_PRIORx_ID`` is set to zero, the compasses will be moved up contiguously, in order, to fill that priority slot on the next reboot. This is easily done in Mission Planner with the arrows on the right side.
```
Comment (bugobliterator) [inline common/source/docs/common-compass-setup-advanced.rst:51]: COMPASS_PRIOx_ID not COMPASS_PRIORx_ID

**Review by bugobliterator — CHANGES_REQUESTED** [review]
Generally looks good to me. Just need to fix the parameter names.

File: common/source/docs/common-compass-setup-advanced.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-compass-setup-advanced.rst]: fixed

File: common/source/docs/common-compass-setup-advanced.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-compass-setup-advanced.rst]: fixed

**Comment by Hwurzburg** [comment]
thanks Sid...merging...

**Pull request merged**